resources/companies/wt.csv

```diff
@@ -0,0 +1,97 @@
+Adres,Kod kraju,Koordynaty,Znaleziony adres,Nazwa firmy,Miejscowość,Kierunek,Tagi,Strona internetowa,Email,Telefon,Płatne?
+"Najświętszej Marii Panny 5F/5, 59-220 Legnica",pl,"51.20872689999999,16.1637251","Najświętszej Marii Panny 5F/5, 59-220 Legnica, Poland",Blumilk sp. z o.o.,Legnica,Informatyka,"Programowanie, Web, PHP, Laravel, Vue",https://www.blumilk.pl/,hello@blumilk.pl,,Nie
+"ul. Młodych Energetyków 12, Bogatynia",pl,"50.9456692,14.9081164","Młodych Energetyków 12, 59-916 Bogatynia, Poland",PGE Elektrownia Turów Spółka Akcyjna,Bogatynia,Informatyka,IT,https://elturow.pgegiek.pl/, elektrowniaturow.giekelt@gkpge.pl,75 773 49 00,Nie
+"ul. Górników Turowa 1, 59-916 Bogatynia",pl,"50.92941889999999,14.9247144","Górników Turowa 1, 59-916 Bogatynia, Poland",BOT Kopalnia Węgla Brunatnego Turów SA,Bogatynia,Informatyka,IT,https://kwbturow.pgegiek.pl/,kopalnia.giekkwbt@gkpge.pl,75 77 35 300,Nie
+"Gałczyńskiego 51, 59-700 Bolesławiec",pl,"51.2645691,15.5892397","Gałczyńskiego 51, 59-700 Bolesławiec, Poland",Zakład Energetyki Cieplnej Sp. z o.o.,Bolesławiec,Informatyka,IT,https://www.zec.boleslawiec.pl/,sekretariat@zec.boleslawiec.pl,"75 732 08 43
+",Nie
```

Comment:
This `",Nie` should be probably in line above. I can see that file has some weird line wrapping; you could fix that.

Reply:
I have no idea why those new lines appeared, as data doesn't seem to be different compared to different cells. I changed export setting to recognize each cell as string.